wordpress-mobile/gutenberg-mobile
# Fix blocks onMerging and onChange red screen errors

This PR makes sure the block is still available when calling an action over it from the native side.

If you type into a block, and than delete a block very fast via the toolbar, the block is not there anymore when the `onChange` call is received from the native side, and the JS code throws the error screen, since there weren't any checks in place to ensure block validity

Same happens `onMerge`, if you type and the fast merge a block.

Companion PR on GB is here https://github.com/WordPress/gutenberg/pull/11537
that removes the delay on `onChange` and makes sure the correct content is used on change, merging, or splitting.

**To test**
- Change the content of a block
- Delete the block immediately
- You should not see the Red Screen

Another test is with merging:
- Try to add/remove/merge block(s)
- Even if the content of the merged block (s) is not correct, the app should not crash
